downstream/modules/automation-hub/proc-configure-ansible-galaxy-cli-verify.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc-configure-ansible-galaxy-cli-verify_{context}"]
+
+= Configuring Ansible-Galaxy CLI to verify collections
+
+You can configure Ansible-Galaxy CLI to verify collections. This assures that collections you download are approved by your organization and have not been changed after it was uploaded to  {HubName}.
+
+.Prerequisites
+
+* Signed and certified collections have been published to {HubName}.
```

Comment:
Certified collections published to hub won't be a pre requisite here, you just want to have signed collections in hub to verify signature, and certified collections can also be signed.